synfig-studio/src/gui/trees/layertree.cpp

```diff
@@ -392,6 +399,13 @@ LayerTree::create_param_tree()
 	//column_time_track->set_visible(false);
 }
 
+void
+LayerTree::set_z_depth_cell_data(Gtk::CellRenderer *cell, const Gtk::TreeIter& it)
```

Comment:
Circle parameter `Feather` currently displays incorrectly:

Before:
![Screenshot_92](https://user-images.githubusercontent.com/5604544/186059904-6ac096ce-4c99-40ac-b97e-50c08bcacf21.png)

Now:
![Screenshot_91](https://user-images.githubusercontent.com/5604544/186059928-b2946db4-002f-4b1d-a12a-4d05c5a934cb.png)

Reply:
oh, I didnt know that the decimal point could be a comma (and now I'm assuming maybe also others), I'll look into it

Reply:
ok, I think now it should be fixed, instead of checking for '.' as the decimal point i used the static function used to return the decimal character, but I couldn't test it as I cant find where to change the deximal point (*_*), will update if i find it.